p2p/switch_test.go

```diff
@@ -4,9 +4,9 @@ import (
 	"bytes"
 	"errors"
 	"fmt"
+	net2 "github.com/line/ostracon/libs/net"
```

Comment:
Thanks for pointing that out. I'll fix it. I think the `goimports` checks don't work in PR? In such an insertion by auto-collect behavior, I'll fix them when I notice, otherwise I may forget about them.

Currently, `make format` makes a lot of unrelated modifications regarding the import order so it cannot use casually. How about creating a PI that commits only the result of `make format`?

Reply:
> Currently, make format makes a lot of unrelated modifications regarding the import order so it cannot use casually. How about creating a PR that commits only the result of make format?

Sounds good 👍